admin/formatter/class-metabox-formatter.php

```diff
@@ -209,24 +201,23 @@ private function get_defaults() {
 			'wincherAutoAddKeyphrases'           => WPSEO_Options::get( 'wincher_automatically_add_keyphrases', false ),
 			'wordproofIntegrationActive'         => YoastSEO()->helpers->wordproof->is_active() ? 1 : 0,
 			'multilingualPluginActive'           => $this->multilingual_plugin_active(),
-
-			/**
-			 * Filter to determine whether the PreviouslyUsedKeyword assessment should run.
-			 *
-			 * @param bool $previouslyUsedKeywordActive Whether the PreviouslyUsedKeyword assessment should run.
-			 */
-			'previouslyUsedKeywordActive'        => apply_filters( 'wpseo_previously_used_keyword_active', true ),
 			'getJetpackBoostPrePublishLink'      => WPSEO_Shortlinker::get( 'https://yoa.st/jetpack-boost-get-prepublish?domain=' . $host ),
 			'upgradeJetpackBoostPrePublishLink'  => WPSEO_Shortlinker::get( 'https://yoa.st/jetpack-boost-upgrade-prepublish?domain=' . $host ),
 			'woocommerceUpsellSchemaLink'        => WPSEO_Shortlinker::get( 'https://yoa.st/product-schema-metabox' ),
 			'woocommerceUpsellGooglePreviewLink' => WPSEO_Shortlinker::get( 'https://yoa.st/product-google-preview-metabox' ),
 		];
+
+		$enabled_features_repo = YoastSEO()->classes->get( Enabled_Features_Repository::class );
```

Comment:
I don't think I want to add a PHPStorm specific docblock for the sake of `ctrl` clicking instead of just searching for the class. Also seeing that we will not look at this code often enough to need to update this class.